.github/Contribution Guide.md

```diff
@@ -1,13 +1,24 @@
 * To contribute to this project first and foremost you should join [this discord](https://discord.gg/9Yh7cWZ). After joining use the #chose-a-role channel to give yourself the roles you wish and in time an admin will give you the relevant roles to give you access to the development section.
-* Next Fork this repository so you have your own instance in your own personal space.
-* *If you want to create a local version on your pc then clone the master branch of your own fork at this point".
-* Create a new branch with the name: "issue-issue_number", replacing issue_number with the issue number you're resolving. For example if I was creating a branch for the issue with number "#5" and name "This is an issue" my branch would be named "This is an issue - #5".
+* Next, fork this repository so you have your own instance in your own personal space.
+* This is done by clicking the Fork button on the top right of the [main repository](https://imgur.com/a/w1I0JiQ)
+* *If you want to create a local version on your pc then clone the master branch of your own fork at this point". This is recommended to be done inside the GitHub desktop client.
+* Create a new branch with the name: "issue-#issue_number", replacing issue_number with the issue number you're resolving. For example if I was creating a branch for the issue with number "5" and name "This is an issue" my branch would be named "This is an issue - #5". 
 * Make your changes to this branch(see below) and commit your changes often with concise but descriptive commit messages referencing the issue("fix #5 add strip movement functionality")
-* **As far as the actual changes go. Please refer to this guide for documenting and structuring the code.** *(This guide is still to be added. Once I've actually started implementing and I(Adam) have found a good code convention to follow I'll create a guide.)*
-* Add your changes to ChangelogInitial.md [Which can be found in: .github/ChangelogInitial.md](https://github.com/JonathanFerguson1/Strips/blob/master/.github/ChangelogInitial.md). These changes will be checked and added to CHANGELOG.md as required before any release.
-* *Push the commits to your fork if you are on a desktop version*
-* Submit a pull request comparing the branch you have made the changes to with the main master branch(on the main repository).
-* Use 1 pull request per issue. This makes it easier to review and keeps everything tidy on the project board.
+* **As far as the actual changes go. Please refer to this guide for documenting and structuring the code.** 
+* Firstly make sure that your master branch is the same version and is uptodate with the master branch of the main repo 
+* Once your changed have been commited to your master branch in your fork. Make a pull request that compares your branch for the issue with the master branch of the main repo. This should mean 1 pull request per issue. And make the request at the beginning so we can see every commit made.
```

Comment:
I would suggest a step creating a pull request before even mentioning making changes. Maybe put that after creating the branch